tests/diceRoller.test.js

```diff
@@ -0,0 +1,58 @@
+const expect = require('chai').expect;
+const assert = require('assert');
+const { rollDice, parseDiceCommand } = require('../commands/diceRoller').forTestsOnly;
+const { register } = require('../commands/diceRoller');
+const { MockBot } = require('./helpers/mockBot');
+const { stripFltSpecialChars } = require('../lib/utils');
+
+describe('diceRoller', function () {
+  describe('#rollDice()', function () {
+    it('should return a number', function () {
+      let results = rollDice({ amount: 1, sides: 4, reroll: true });
+      expect(results).to.be.a('number');
+    });
+    it('should never return a 1 if reroll is true', function () {
```

Comment:
I often find it useful to put new lines between `it` blocks in tests. This helps readability, and selfishly it makes the `{` and `}` bindings to jump between blank lines in Vim-mode very useful.